client/src/pages/dashboard/views/Staff.svelte

```diff
@@ -1,12 +1,88 @@
 <script lang="ts">
     import { dashboardState } from '../stores/DashboardState';
+    import { staffList } from '../stores/StaffStore';
+    import { allOffices } from '../stores/OfficeStore';
+
+    import { IconSize, PersonPayload, RowType, Events } from '../../../components/types';
+    import PersonRowLocal from '../../../components/ui/itemrow/PersonRowLocal.svelte';
+    import LocalPermissions from '../../../components/ui/forms/permissions/LocalPermissions.svelte';
+    import RemoveStaff from '../../../components/ui/forms/staff/RemoveStaff.svelte';
+    import Modal from '../../../components/ui/Modal.svelte';
+    import PersonContext from '../../../components/ui/contextdrawer/PersonContext.svelte';
 
     $: ({ currentOffice } = $dashboardState);
+    $: officeName = currentOffice === null ? 'No office name.' : $allOffices[currentOffice];
+
+    let showContextMenu = false;
+    let showLocalPermission = false;
+    let showRemoveStaff = false;
+    let currentContext = null as PersonPayload | null;
+
+    function overflowClickHandler(e: CustomEvent<PersonPayload>) {
+        currentContext = e.detail;
+        showContextMenu = true;
+    }
+
+    function contextMenuHandler(e: CustomEvent<PersonPayload>) {
+        switch (e.type) {
+            case Events.EditLocalPermission:
+                showLocalPermission = true;
+                break;
+            case Events.RemoveStaff:
+                showRemoveStaff = true;
+                break;
+            default: break;
+        }
+    }
 </script>
 
 {#if currentOffice === null}
     You must select an office before accessing the Staff page.
 {:else}
-    Staff page of Office ID {currentOffice}.
-    <h1>Staff</h1>
+    {#await staffList.load()}
+        <p>Loading staff page...</p>
+    {:then}
+        <h1>Staffs of {officeName}</h1>
+        {#if staffList === null}

```

Comment:
You probably meant to extract the store here?
```suggestion
        {#if $staffList === null}
```